### DISCUSSION TITLE
i'd like to suggest a "Dice" icon for randomization purposes.

### DISCUSSION THREAD

**Discussion by kajloevesijn** [tailwindlabs/heroicons#1007] [Ideas]
basically the title, a dice icon is nearly always used as a means to communicate something that will trigger some kind of randomization, wether it's a seed, name for an object or a color.

**Comment by tgf9** [comment]
An icon to represent "randomness" has been proposed before.

- https://github.com/tailwindlabs/heroicons/discussions/370
- https://github.com/tailwindlabs/heroicons/discussions/534

I would also like to see an icon for random. :smiley:

**Comment by kajloevesijn** [comment]
A shuffle icon would be really good to have, although I feel it doesn't represent random generators very well. A shuffle icon in my eyes represents the randomization of existing elements in a list.

**Comment by ndri** [comment]
I added `dice` and `arrows-crossing` (for shuffle) to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here:
- https://sidekickicons.com/?icon=dice
- https://sidekickicons.com/?icon=arrows-crossing
